force-app/main/default/lwc/nksFamilyViewerEntry/nksFamilyViewerEntry.js

```diff
@@ -302,14 +302,22 @@ export default class nksFamilyViewerEntry extends NavigationMixin(LightningEleme
     }
 
     getResponsibilityParentText() {
+        let res = '';
+
+        if (this.relation.responsible === '' || this.relation.responsible === null) {
+            return '';
```

Comment:
Burde det ikke stå noe tekst her, kanskje "Vet ikke foreldreansvar" eller "Fant ikke foreldreansvar"? Det er veldig dumt å skjule informasjon for veilederne, spesielt hvis de venter på den og så kommer den aldri

Reply:
når responsible er null betyr at foreldreansvar listen for bruker er tom, ikke at det finnes men vi fant ikke altså vi skjuler ikke informasjon

Reply:
Tror Morten mener at vi bare bør vise en placeholder/default verdi hvis det vanligvis er en plass hvor en tekst/data vises, sånn at de ikke tror at den bare bruker lang tid på å laste inn e.l. og sitter og venter på den

Reply:
Ok, jeg kan høre med Andreas/Sara om de ønsker en tekst der